setting-rate-limit-cloud-api.html.md.erb

```diff
@@ -5,11 +5,47 @@ owner: CAPI
 
 Cloud Foundry lets you set rate limits on the number of requests third parties can make to the Cloud Controller API. You can set different rate limits for authenticated and unauthenticated users.
 
-Use the following properties in the cloud_controller BOSH job to define rate limits:
-
-```
-cc.rate_limiter.enabled
-cc.rate_limiter.general_limit
-cc.rate_limiter.unauthenticated_limit
-cc.rate_limiter.reset_interval_in_minutes
-```
\ No newline at end of file
+When rate limiting is enabled responses to API calls [contain information](./rate-limit-cloud-controller-api.html) about the limit, how close the user is to reaching it, and when it will be reset.
+
+Use the following properties in the `cloud_controller_ng` BOSH job to define rate limits:
+
+<table>
+<th>Property</th>
+<th>Description</th>
+<th>Default value</th>
+ <tr>
+	<td>cc.rate_limiter.enabled</td>
+	<td>Enable rate limiting for UAA-authenticated endpoints per user or client</td>
+    <td>false</td>
+ </tr>
+ <tr>
+	<td>cc.rate_limiter.general_limit</td>
+	<td>The number of requests a user or client is allowed to make for all endpoints that do not have a custom limit over the configured interval</td>
+    <td>2000</td>
+ </tr>
+ <tr>
+	<td>cc.rate_limiter.unauthenticated_limit</td>
+	<td>The number of requests an unauthenticated client is allowed to make over the configured interval</td>
+    <td>100</td>
+ </tr>
+ <tr>
+	<td>cc.rate_limiter.reset_interval_in_minutes</td>
+	<td>The interval in minutes after which a user's available API requests will be reset</td>
+    <td>60</td>
+ </tr>
+ <tr>
+    <td>cc.nginx_rate_limit_general</td>
+    <td>The rate limiting and burst value to use for '/'</td>
+    <td>(none)</td>
+ </tr>
+ <tr>
+    <td>cc.nginx_rate_limit_zones</td>
+    <td>Array of zones to do rate limiting for.</td>
```

Comment:
I dont know what zones we reference here. A Hyperscaler Availabillity Zone ? Also does this just then apply to the root endpoint as the naming of that variable suggests its just there for cc.nginx_rate_limit_general.

Reply:
I would not add documentation on the nginx rate limiting. We have no experience with it.

Reply:
Are you sure? I'm just reproducing what is already documented in the job's spec file (for the nginx bits it's [here](https://github.com/cloudfoundry/capi-release/blob/develop/jobs/cloud_controller_ng/spec#L877-#L892))

Reply:
I'm sure that we have no experience with nginx rate limits :-)

Reply:
Ok I'll cut it out now